src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -24,24 +24,26 @@ public class OrderController {
     private final ResponseDtoMapper<OrderResponseDto, Order> orderResponseDtoMapper;
 
     public OrderController(ShoppingCartService shoppingCartService,
-                           OrderService orderService,
-                           UserService userService,
-                           ResponseDtoMapper<OrderResponseDto, Order> orderResponseDtoMapper) {
+            OrderService orderService,
+            UserService userService,
+            ResponseDtoMapper<OrderResponseDto, Order> orderResponseDtoMapper) {
         this.shoppingCartService = shoppingCartService;
         this.orderService = orderService;
         this.userService = userService;
         this.orderResponseDtoMapper = orderResponseDtoMapper;
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        ShoppingCart cart = shoppingCartService.getByUser(
+                userService.findByEmail(authentication.getName()).get());
```

Comment:
let's throw an exception in case user is not found, the same for other similar cases